### DISCUSSION TITLE
"# meetings in progress" - time zone issue? TMSL UI

### DISCUSSION THREAD

**Discussion by toddjsaunders** [code4recovery/12-step-meeting-list#692] [Q&A] [answered]
Hello,

We're having an issue where the "# of meetings in progress" is showing for meetings 3 hours in advance of when they actually start.

For example, it's 3:15 (PST) now, and it's showing 2 meetings that don't start until 5:30pm currently in progress.

I have the address set correctly on the in-person meeting, and the other that's showing up is online, and I have the approximate location set correctly.

I believe I've seen some using UI where this feature is disabled, that would also be an option for us, I just can't figure out how to do it :)

Thanks!

**Comment by joshreisner** [comment, marked as answer]
check the timezone in wordpress settings

**Reply by toddjsaunders** [reply]
Thanks much Josh - that did it.
